hackathon_site/event/tests.py

```diff
@@ -786,12 +786,6 @@ def test_serializer(self):
         }
         self.assertEqual(profile_expected, profile_serialized)
 
-    def test_readonly_serializer_fields(self):
```

Comment:
i removed these tests to accomodate the changed serializer, other functionality not affected.